YoutubeDownloader/Converters/VideoQualityPreferenceToStringConverter.cs

```diff
@@ -1,23 +1,22 @@
 ﻿using System;
 using System.Globalization;
-using System.Windows.Data;
+using Avalonia.Data.Converters;
 using YoutubeDownloader.Core.Downloading;
 
 namespace YoutubeDownloader.Converters;
 
-[ValueConversion(typeof(VideoQualityPreference), typeof(string))]
 public class VideoQualityPreferenceToStringConverter : IValueConverter
 {
     public static VideoQualityPreferenceToStringConverter Instance { get; } = new();
 
-    public object? Convert(object? value, Type targetType, object parameter, CultureInfo culture)
+    public object? Convert(object? value, Type targetType, object? parameter, CultureInfo culture)
     {
         if (value is VideoQualityPreference preference)
             return preference.GetDisplayName();
```

Comment:
By the way, is there a way to streamline this in Avalonia?

Can you do something like this in Avalonia bindings: `Text={Binding VideoQuality.GetDisplayName()}`, where `GetDisplayName()` is an extension method? It wouldn't have to watch for `INotifyPropertyChanged`, just pull the value once.

Reply:
No, sadly not, Avalonia has compiled bindings, but not in the x:Bind way where you can call methods (at least not yet).